src/backend/EX/settings.py

```diff
@@ -186,3 +188,83 @@
 # Other
 LOGIN_REDIRECT_URL = 'home'
 ACCOUNT_LOGOUT_REDIRECT_URL = 'home'
+
+# Ability Constants
+VALID_ABILITY_KEYS = {
```

Comment:
If the key in the HashTable always equals the value, we can use HashSet instead.

Reply:
Using a dictionary allows u utilize the hint in IDE for better type support, rather hash set needs additional step for retrieving a key.